designs/2020-init-command-eslint-cli/README.md

```diff
@@ -60,8 +89,9 @@ This might increase maintenance burden and if we are planing to have a monorepo,
     change for them? If so, how are you going to minimize the disruption
     to existing users?
 -->
-Releasing with a proper message which the command is being run from `eslint` with something like - `"this command has been moved from here, use 'npx @eslint/create'"` might be a good way to address backward compatibility. 
-Otherwise we can keep the feature in the main repo as well for starting few days/weeks with a warning and then removing it completing with an error message. 
+- Releasing with a proper message which the command is being run from `eslint` with something like - `"this command has been moved from here, use 'npx @eslint/create'"` might be a good way to address backward compatibility. 
+- Or whenever `--init` command is being used, show a warning and run `npx @eslint/create` using `child_process` of the native node modules. 
```

Comment:
I think we can just run `npx @eslint/create` directly without a warning message. We'll still get the benefit of not including all of the init functionality in the main package and we'd minimize the chance of problems if someone came across some old documentation. At most we may want to just output a message like "You can also run this command directly using npm init @eslint".

Reply:
That works too. We should use whatever we end up documenting in the getting started guide.